composer.json

```diff
@@ -23,71 +22,73 @@
         "doctrine/doctrine-migrations-bundle": "^1.1",
         "doctrine/migrations": "~1.4",
         "doctrine/orm": "~2.5",
-        "fortawesome/font-awesome": "~4.2.0",
         "graylog2/gelf-php": "^1.5",
         "incenteev/composer-parameter-handler": "~2.0",
-        "jms/translation-bundle": "~1.3.0",
-        "malukenho/docheader": "^0.1.7",
-        "mopa/bootstrap-bundle": "^3.2",
-        "mopa/composer-bridge": "~1.5",
-        "nelmio/security-bundle": "~1.4",
-        "openconext/monitor-bundle": "^1.0",
-        "sensio/distribution-bundle": "^5.0",
-        "sensio/framework-extra-bundle": "~3.0",
+        "jms/translation-bundle": "^1.3.0",
+        "nelmio/security-bundle": "^2",
+        "openconext/monitor-bundle": "^2.0",
+        "sensio/framework-extra-bundle": "^5.0",
         "surfnet/messagebird-api-client-bundle": "^4.0",
         "surfnet/stepup-bundle": "^4.0",
         "surfnet/stepup-saml-bundle": "^4.1",
-        "surfnet/stepup-u2f-bundle": "^3.4",
-        "surfnet/yubikey-api-client-bundle": "^3.0",
-        "symfony/assetic-bundle": "~2.3",
+        "surfnet/stepup-u2f-bundle": "^4",
```

Comment:
Can the u2f bundle be removed from this project too?

Reply:
There is another PR for that: https://github.com/OpenConext/Stepup-Gateway/pull/182 Since its quite some work and not finished yet. I did not integrate it with this one.